ci/travis-36-numpydev.yaml

```diff
@@ -0,0 +1,18 @@
+name: pandas
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python=3.6*
+  - pytz
+  - Cython
+  # universal
+  - pytest
+  - pytest-xdist
+  - moto
+  - pip:
+    - "git+git://github.com/dateutil/dateutil.git"
+    - "-f https://7933911d6844c6c53a7d-47bd50c35cd79bd838daf386af554a83.ssl.cf2.rackcdn.com"
```

Comment:
this actually works in this syntax?

Reply:
Surprisingly, yes! This is valid syntax for requirements.txt, which is all that conda is doing.